ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/fragmentPage/v1/fragmentPage/headerlibs.html

```diff
@@ -0,0 +1,23 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Even this is not required.

Reply:
Yes, this will be removed as this was for static implementation.